geode-docs/managing/disk_storage/backup_restore_disk_store.html.md.erb

```diff
@@ -38,6 +38,15 @@ Do not try to create backup files from a running system by using your operating
 **Preparing to Make a Backup**
 
 -   Consider compacting your disk store before making a backup. If auto-compaction is turned off, you may want to do a manual compaction to save on the quantity of data copied over the network by the backup. For more information on configuring a manual compaction, see [Manual Compaction](compacting_disk_stores.html#compacting_disk_stores__li_63CF8C35153D4173AADF7DC35FEC61F9).
+-   Take the backup when region operations are quiescent,
+to avoid the possibility of an inconsistency between region data and
+an asyncronous event queue (AEQ),
+if that backup were to be used to restore a system.
+The AEQ is persistent.
+If a backup is unlucky enough to be taken between
```

Comment:
Instead of "If the backup is unlucky enough to be taken between", consider:
If the timing of the backup occurs between...